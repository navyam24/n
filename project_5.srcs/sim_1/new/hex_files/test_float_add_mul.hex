3f9df0b7
3b608093
40a00137
00010113
002081d3
0001805b
3f9df0b7
3b608093
c0a00137
00010113
002081d3
0001805b
449a40b7
00008093
c59c4137
00010113
002081d3
0001805b
447a00b7
00008093
bf000137
00010113
002081d3
0001805b
3f9df0b7
3b608093
40a00137
00010113
102081d3
0001805b
3f9df0b7
3b608093
c0a00137
00010113
102081d3
0001805b
449a40b7
00008093
c59c4137
00010113
102081d3
0001805b
447a00b7
00008093
bf000137
00010113
102081d3
0001805b
0000007b
