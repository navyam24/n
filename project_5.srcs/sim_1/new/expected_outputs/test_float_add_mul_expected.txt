out.s 0 01000000110001110111110011101101 6.234000
out.s 1 11000000011100010000011000100110 -3.766000
out.s 2 11000101011010110110000000000000 -3766.000000
out.s 3 01000100011110011110000000000000 999.500000
out.s 4 01000000110001010111000010100011 6.170000
out.s 5 11000000110001010111000010100011 -6.170000
out.s 6 11001010101111000100101100100000 -6170000.000000
out.s 7 11000011111110100000000000000000 -500.000000
